osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,34 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
+:context: byo-vpc
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured Virtual Private Cloud (VPC) allows for the use of any existing network that relies on an existing VPC.
+
+.Prerequisites
+* You have a configured VPC. Use the following documentation for guidance on how to configure the VPC's components, such as:
+** a private subnet
+** a public subnet
+** a NAT gateway
+** an Internet gateway
+* You have a configured AWS account in your AWS organization.
+
+.Procedure
```

Comment:
There shouldn't be a procedure directly in the assembly. Can you break this out into a separate module and make sure to have an intro about what this procedure is actually doing? It's not clear as is when it jumps right in.

The creating an IAM user steps can be a module and be the first one listed before you get into the two availability zone options